sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -583,12 +583,16 @@ abstract class JdbcDialect extends Serializable with Logging {
    * {@link OracleDialect.OracleSQLQueryBuilder} and
    * {@link MsSqlServerDialect.MsSqlServerSQLQueryBuilder}.
    */
-  def supportsLimit: Boolean = true
+  def supportsLimit: Boolean = false
```

Comment:
This looks like a breaking change for the 3rd party JDBC Dialect. Please revert this.

Reply:
`supportsLimit` is a new API and it is not in branch-3.4

Reply:
Oh, got it~